back/src/main/java/com/ogjg/back/common/exception/ErrorCode.java

```diff
@@ -0,0 +1,28 @@
+package com.ogjg.back.common.exception;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public enum ErrorCode {
+
+    SUCCESS(HttpStatus.OK,"200","OK"),
+    NOT_FOUND_USER(HttpStatus.NOT_FOUND, "코드는내맘대로에러","회원를 찾을 수 없습니다");
```

Comment:
예시지만, 실제로 깃허브에 올라가는 코드이기 때문에 임시로라도 임시 에러 코드를 예시로 넣어보는 것은 어떠신가요?
- `"코드는내맘대로에러"`에서 `"4001"`(임시) 처럼 바꾸면 다른 사람이 보기에 어색하지 않을 것 같습니다!

그리고 `,`뒤에 띄어쓰기와 `"를"` 오타 수정 부탁드립니다!

```suggestion
    NOT_FOUND_USER(HttpStatus.NOT_FOUND, "4001", "회원을 찾을 수 없습니다");
```